processor/src/fast/mod.rs

```diff
@@ -243,11 +240,13 @@ impl FastProcessor {
     /// a | b | c | d | e | f | g | h | i | j | k | l | m | n | o | p
     ///
     /// Then
-    /// - `stack_get_word(0)` returns `[d, c, b, a]`,
-    /// - `stack_get_word(1)` returns `[e, d, c ,b]`,
+    /// - `stack_get_word_be(0)` returns `[d, c, b, a]` (big-endian),
+    /// - `stack_get_word_be(1)` returns `[e, d, c, b]` (big-endian),
     /// - etc.
+    ///
+    /// This matches the behavior of `mem_loadw_be` where `mem[a+3]` ends up on top of the stack.
     #[inline(always)]
-    pub fn stack_get_word(&self, start_idx: usize) -> Word {
+    pub fn stack_get_word_be(&self, start_idx: usize) -> Word {
```

Comment:
@plafer do you think these are necessary in the FastProcessor? The main purpose is to make the external API clearer, but I think this is purely internal.

Reply:
Your call, I'm okay with either. Seems like you already reverted them, so that works too.